bin/build-drivers/build_drivers/install_driver_locally.clj

```diff
@@ -0,0 +1,50 @@
+(ns build-drivers.install-driver-locally
```

Comment:
The verby name reads a bit oddly as an ns name. How about `build-drivers.local-maven` or similar?